pkg/noderesourcetopology/least_numa.go

```diff
@@ -0,0 +1,173 @@
+package noderesourcetopology
+
+import (
+	"fmt"
+
+	topologyv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+	"gonum.org/v1/gonum/stat/combin"
```

Comment:
`combin` should sit in its own group:
```
stdlib

3rd party packages

kube packages
```
(perhaps inverting 3rd party and kube)

Reply:
👍